src/psyclone/psyGen.py

```diff
@@ -1297,12 +1297,10 @@ def list(self, indent=0):
         return result
 
     def list_to_string(self, my_list):
-        result = ""
-        for idx, value in enumerate(my_list):
-            result += str(value)
-            if idx < (len(my_list) - 1):
-                result += ","
-        return result
+        ''':returns: a comma-delimited string of all members.
+        :rtype: str
+        '''
+        return ",".join(my_list)
```

Comment:
Gosh, that must have been an old routine - we know a lot more Python now! Please document the `my_list` argument to the function.

Reply:
I forgot to ask this: I actually wanted to suggest to remove this function, and replace the (iirc) three calls to it with just ",".join() - is that ok?

Reply:
Yes, I nearly suggested that myself.

Reply:
Done.